src/gridstack-dd.ts

```diff
@@ -352,6 +353,11 @@ GridStack.prototype._prepareDragDropByNode = function(node: GridStackNode): Grid
       this._gsEventHandler[event.type](event, target);
     }
 
+    // Saving `minRow` and adding new ones for resizing
+    minRow = this.opts.minRow;
+    this.opts.minRow = this.getRow() + 100;

```

Comment:
big no no - don't arbitrary add 100 rows!
we should only add 1 extra row (when computing the needed rows) IFF the item you are resizing/moving is at the bottom (node.y+node.h). don't need to mod minRow for this...